mmv1/templates/terraform/custom_flatten/redis_cluster_automated_backup_config.go.tmpl

```diff
@@ -0,0 +1,65 @@
+func flatten{{$.GetPrefix}}{{$.TitlelizeProperty}}(v interface{}, d *schema.ResourceData, config *transport_tpg.Config) interface{} {
+	if v == nil {
+		return nil
+	}
+	original := v.(map[string]interface{})
+	if len(original) == 0 {
+		return nil
+	}
+	// if automated_backup_config is not defined 
+    if len(d.Get("automated_backup_config").([]interface{})) < 1 {
```

Comment:
Sorry- can you walk me through why we need to check state again? I think we should be checking for DISABLED in the API response here.